tests/integrations/pd/test_rpc_client.rs

```diff
@@ -131,6 +131,26 @@ fn test_rpc_client() {
     client.scatter_region(region_info).unwrap();
 }
 
+#[test]
+fn test_connect_follower() {
+    let connect_leader_fp = "connect_leader";
+    let server = MockServer::new(2);
+    let eps = server.bind_addrs();
+    let mut cfg = new_config(eps.clone());
+    cfg.enable_forwarding = true;
+    let mgr = Arc::new(SecurityManager::new(&SecurityConfig::default()).unwrap());
+    let client = RpcClient::new(&cfg, None, mgr).unwrap();
+    fail::cfg(connect_leader_fp, "return").unwrap();
+    // RECONNECT_INTERVAL_SEC is 1s.
+    thread::sleep(Duration::from_secs(1));
+    let _ = client.alloc_id();
```

Comment:
Should check id is allocated successfully.

Reply:
I think it's not necessary to check the result